packages/components/addon/components/hds/app-footer/status-link.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export const STATUSES = {
+  OPERATIONAL: {
+    text: 'Operational',
+    iconName: 'check-circle-fill',
+    iconColor: 'var(--token-color-foreground-success)',
+  },
+  DEGRADED: {
+    text: 'Degraded',
+    iconName: 'alert-triangle',
+    iconColor: 'var(--token-color-foreground-warning)',
+  },
+  OUTAGE: {
+    text: 'Outage',
+    iconName: 'x-square-fill',
+    iconColor: 'var(--token-color-foreground-critical)',
+  },
+  MAINTENANCE: {
+    text: 'Maintenance',
+    iconName: 'info-fill',
+    iconColor: 'var(--token-color-foreground-highlight)',
+  },
+};
+
+export default class HdsAppFooterStatusLinkComponent extends Component {
+  get status() {
+    return this.args.status.toUpperCase();
+  }
+
+  get statusIcon() {
+    return STATUSES[this.status.toUpperCase()].iconName;
+  }
+
+  get statusIconColor() {
+    return STATUSES[this.status.toUpperCase()].iconColor;
+  }
+
+  get text() {
+    return this.args.text ?? STATUSES[this.status.toUpperCase()].text;
+  }
+
```

Comment:
Note: Not sure what else would be needed for this component yet to display actual status from a service.

Reply:
Thanks, I exposed them and added examples to the Showcase.